src/js/components/search/filters/keyword/SelectedKeywords.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * SelectedKeywords.jsx
+ * Created by David Trinh on 5/2/18.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import ShownKeyword from './ShownKeyword';
+
+const propTypes = {
+    toggleKeyword: PropTypes.func,
+    selectedKeyword: PropTypes.object
+};
+
+export default class SelectedKeywords extends React.Component {
+    render() {
+        let hideTags = 'hide';
+        if (this.props.selectedKeyword.size !== 0) {
+            hideTags = '';
+        }
+        const shownKeywords = [];
+        this.props.selectedKeyword.forEach((keyword) => {
```

Comment:
This would be a good place to use [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) and combine lines 21 and 22.